mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -577,32 +618,42 @@ def test_read_vmrk_annotations(tmpdir):
 
 
 @testing.requires_testing_data
-def test_read_vhdr_annotations_and_events():
+def test_read_vhdr_annotations_and_events(tmpdir):
     """Test load brainvision annotations and parse them to events."""
+    # First we add a custom event that contains a comma in its description
+    for src, dest in zip((vhdr_path, vmrk_path, eeg_path),
+                         ('test.vhdr', 'test.vmrk', 'test.eeg')):
+        shutil.copyfile(src, tmpdir / dest)
+
+    # Commas are encoded as "\1"
+    with open(tmpdir / 'test.vmrk', 'a') as fout:
+        fout.write(r"Mk15=Comma\1Type,CommaValue\11,7800,1,0")
+        fout.write("\n")
```

Comment:
Why the extra line?

Reply:
to end a file with an empty new line, which is the default on Linux and MacOS, but strictly speaking it would not be necessary in this case.

Reply:
No, I think the `\n` could be combined with the previous line.